### ISSUE TITLE
Bonsai Badge

### ISSUE THREAD

**Issue by asachs01** [sensu/sensu-go-plugin#11]
Similar to #9 , this issue to is get a badge into the repo so that folks know where to download the asset (a la bonsai).

**Comment by asachs01** [comment]
Working badge:
![](https://img.shields.io/badge/Bonsai-Download%20Me-brightgreen.svg?colorB=89C967)

Once Shields updates:
`https://img.shields.io/badge/Bonsai-Download%20Me-brightgreen.svg?colorB=89C967?logo=sensu`

We should then be able to do something like:
```
[![Bonsai Asset - CHANGEME](https://img.shields.io/badge/Bonsai-Download%20Me-brightgreen.svg?colorB=89C967?logo=sensu)](https://bonsai.sensu.io/assets/CHANGEME/CHANGEME)
```

**Comment by asachs01** [comment]
Badges are working!

![](https://img.shields.io/badge/Bonsai-Download%20Me-brightgreen.svg?colorB=89C967&logo=sensu)

I'll put in a PR to add this

**Comment by apaskulin** [comment]
Closed by #13
